src/test/ij.c

```diff
@@ -1561,6 +1561,19 @@ main( hypre_int argc,
          arg_index++;
          snprintf(mem_tracker_name, HYPRE_MAX_FILE_NAME_LEN, "%s", argv[arg_index++]);
       }
+#endif
+#if defined(HYPRE_USING_NODE_AWARE_MPI)
+      else if ( strcmp(argv[arg_index], "-node_aware_lvl_threshold") == 0 )
+      {
+         arg_index++;
+         hypre_HandleNodeAwareSwitchoverThreshold(hypre_handle()) = atoi(argv[arg_index++]);
+         if (hypre_HandleNodeAwareSwitchoverThreshold(hypre_handle()) > 0) {
```

Comment:
Should `hypre_HandleNodeAwareSwitchoverThreshold` be an option internal to BoomerAMG instead of a global variable?

Reply:
This was a first approach to have the value accessible. I'm not sure how we could have it as part of BoomerAMG but yes it should only be used for BoomerAMG.

Reply:
Thank you! @liruipeng, do you support the idea of moving this switch threshold inside BoomerAMG?

Reply:
Hi @victorapm I totally agreed with you. It would be much better if we can move it into the AMG struct. But a technical issue is that we need to access this value in some functions involved in the solve phase but the AMG struct is not passed along to those functions. So we ended up having a global variable for its simplicity. Any suggestions? Thanks.

Reply:
@liruipeng thanks for providing the context. Now, I can see it better! See my follow-up comments at https://github.com/hypre-space/hypre/pull/941#discussion_r1331547752 and https://github.com/hypre-space/hypre/pull/941#discussion_r1331548539
Sorry to bug with this, but I think it's better if we have less global variables